server/index.js

```diff
@@ -1,10 +1,13 @@
+/* eslint-disable no-unused-vars */
+
 require('dotenv/config');
 const express = require('express');
 
 const db = require('./database');
 const ClientError = require('./client-error');
 const staticMiddleware = require('./static-middleware');
 const sessionMiddleware = require('./session-middleware');
+const { query } = require('express');
```

Comment:
I don't think you meant to `require()` this.

Reply:
Not sure how it popped up in there.  That is way I disabled unused vars lol.  I think I accidently typed "query" as a variable and it showed up after.